index.html

```diff
@@ -29,25 +50,27 @@
 </head>
 
 <body>
-    <div>Header</div>
-    <div>
-       <!--
+    <h1 class="mx-auto" style="width: 200px;">Calculator</h1>
```

Comment:
This style attribute use is good. Note that a good improvement is to move the styling into css named classes. For example class="mx-auto calc" could be in place here and then in the style tag you can add

```
.calc {
  width: 200px;
}
```